packages/block-library/src/post-terms/edit.js

```diff
@@ -94,6 +94,16 @@ export default function PostTermsEdit( {
 					} }
 					help={ __( 'Enter character(s) used to separate terms.' ) }
 				/>
+				<ToggleControl
+					label={ __( 'no link' ) }
+					checked={ noLink }
+					onChange={ ( nextValue ) =>
+						setAttributes( { noLink: nextValue } )
+					}
+					help={ __(
+						'Toggle on the option to remove the terms links from the output.'
```

Comment:
I am not sure it needs a help text, but if it does, I think this text is potentially difficult to understand, it is very technical.

Reply:
Sure, let's figure out a better phrasing for the hint, I am not a native English speaker, so would probably make it read worse :). Can you suggest something more appropriate, or, as you said, we can simply remove it? Please advise. Thanks!